src/custom/state/orders/updater.ts

```diff
@@ -46,7 +29,7 @@ function _computeFulfilledSummary({
 
   // if we can find the order from the API
   // and our specific order exists in our state, let's use that
-  if (orderFromApi && Number(orderFromApi.executedBuyAmount) > 0 && Number(orderFromApi.executedSellAmount) > 0) {
+  if (orderFromApi && isOrderFinalized(orderFromApi)) {
```

Comment:
nit: do you need the initial `orderFromApi` since `isOrderFinalised` accepts `null` and would return `false`?

Reply:
Because TS.
`orderFromApi` type is `MetaOrderWhatever | null`

The function alone is not enough to type guard `orderFromApi` to the type. And when I use it later TS complains.
Or at least I don't know how to do that.
Suggestions?